circle.yml

```diff
@@ -51,10 +51,12 @@ defaults:
       # git fetch origin evm2wasm-trace
       # git checkout evm2wasm-trace
       command: |
-        git clone https://github.com/ethereum/cpp-ethereum --branch ewasm-json-trace --single-branch
+        git clone https://github.com/ethereum/cpp-ethereum --branch develop --single-branch
```

Comment:
Should be using the `ewasm` branch IIRC.

Reply:
`ewasm` branch last commit was 23 days ago. I've been using `develop` locally and it works.

Reply:
Just rebase `ewasm` against `develop` and push it. The last decision was it is kept up to date. However I do not think there were any relevant changes on `develop`.